railties/lib/rails/commands/plugin/plugin_command.rb

```diff
@@ -25,7 +25,7 @@ def perform(type = nil, *plugin_args)
           railsrc = File.expand_path(options[:rc])
 
           if File.exist?(railsrc)
-            extra_args = File.read(railsrc).split(/\n+/).flat_map(&:split)
+            extra_args = File.read(railsrc).gsub(/#.*\n/, "\n").split(/\n+/).flat_map(&:split)
```

Comment:
```suggestion
            extra_args = File.read(railsrc).gsub(/^\s*#.*$/, "\n").split(/\n+/).flat_map(&:split)
```

The current Regexp doesn't capture properly the comments

Reply:
Could you provide an example that does not work with the original regex `gsub(/#.*\n/, "\n")`?